File: easy/ruby/letter_capitalize.rb
# Have the function LetterCapitalize(str) take the str parameter being passed 
# and capitalize the first letter of each word. Words will be separated by 
# only one space.
def LetterCapitalize(str)
  str = str.scan(/\w+/)
    
  str.each do |word|
    word.capitalize!
  end
              
  return str.join(" ")       
end

# keep this function call here
LetterCapitalize(STDIN.gets)

File: easy/LetterCapitalize.js
function LetterCapitalize(str) {
  var words = str.split(' ');

  for(var i = 0; i < words.length; i++) {
    words[i] = words[i][0].toUpperCase() + words[i].substring(1);
  }

  return words.join(' ');
}

LetterCapitalize(readline());
